lib/systems/include/Controllers/CASEControllerWrapper.h

```diff
@@ -5,14 +5,18 @@
 #include "CASESystem.h"
 
 template <typename message_queue>
+/// @brief makes CASE system a part of Controller hierarchy for use in TC Mux
 class TorqueControllerCASEWrapper : public virtual Controller
 {
 public:
     TorqueControllerCASEWrapper() = delete;
-    
+    /// @param case_instance provide current instance to ensure there are not duplicates of this system as it needs to stay updated for MCU to get data.
```

Comment:
Please update this comment to be more immediately-readable. Is "provide current instance" a direction to a user? I shouldn't have to read it twice to understand what's being said.

Reply:
It's very uncommon to see documentation direct the user to do anything. Instead, specify that this function "requires" some particular inputs.